pynecone/utils.py

```diff
@@ -298,6 +299,44 @@ def get_config() -> Config:
         return Config(app_name="")
 
 
+def check_node_version(min_version):
+    """Check the version of Node.js.
+
+    Args:
+        min_version: The minimum version of Node.js required.
+
+    Returns:
+        Whether the version of Node.js is high enough.
+    """
+    try:
+        # Run the node -v command and capture the output
+        result = subprocess.run(
+            ["node", "-v"], stdout=subprocess.PIPE, stderr=subprocess.PIPE
+        )
+        # The output will be in the form "vX.Y.Z", so we can split it on the "v" character and take the second part
+        version = result.stdout.decode().strip().split("v")[1]
```

Comment:
I don't think we need this line either if we don't cast to int

Reply:
I actually need this line because the output has v